charms/kfp-persistence/src/components/relation_components.py

```diff
@@ -0,0 +1,127 @@
+import logging
```

Comment:
Let's not forget to use the common component Andrew just pushed.

Reply:
Done.